docs/source/contributing/developing.adoc

```diff
@@ -192,12 +192,19 @@ $go test -v <relative path of package> -run "Test<Regex pattern to match tests>"
 For more information about test options, run the `go test --help` command and review the documentation.
 
 [[integration-tests]]
-=== Running Integration Tests
+=== Integration Tests
```

Comment:
I did small changes here, checked by Hardy and Budhram already, but could I please ask you for an additional quick check, @thatdocslady?